ts/src/test/test.ts

```diff
@@ -794,13 +794,13 @@ export default class testMainClass extends baseMainTestClass {
         // to make this test as fast as possible
         // and basically independent from the exchange
         // so we can run it offline
-        const filename = './ts/src/test/static/markets/' + id + '.json';
+        const filename = rootDir + './ts/src/test/static/markets/' + id + '.json';
```

Comment:
@ttodua both static tests and broker Id tests are running on master successfully 🤔  how are you running them?

Reply:
`python/ccxt/test/test_async.py exchangeName ... etc..`

Reply:
@ttodua if I run that it works:

```
python python/ccxt/test/test_async.py binance --static
Exchange to test: binance
[PY][TEST_SUCCESS] 11 static tests passed.
```

Reply:
```
 python python/ccxt/test/test_async.py --idTests       
[PY][TEST_SUCCESS] brokerId tests passed.
```

Reply:
well strange ... https://imgsh.net/a/aaqqIpb.png
happens in debugger, but not from cli

Reply:
so, it does only work when `python` is called from `ccxt` dir, otherwise it errors. 
I still think that such relative path are fragile and we could have replaced them with full reference path, which works in other cases too. 
as you decide

Reply:
Yes if you call the file from a different root other than ccxt/ that relative path will not work


Sure, if the build succeeds here we can merge it.